build.gradle.kts

```diff
@@ -94,6 +94,18 @@ val runWithCustomSandbox by intellijPlatformTesting.runIde.registering {
    }
 }
 
+//tasks.named<RunIdeTask>("runIde") {
+//   jvmArgumentProviders += CommandLineArgumentProvider {
+//      listOf("-Didea.kotlin.plugin.use.k2=true")
+//   }
+//}
+
+//tasks.test {
+//   jvmArgumentProviders += CommandLineArgumentProvider {
+//      listOf("-Didea.kotlin.plugin.use.k2=true")
+//   }
+//}
+
```

Comment:
I believe we should not commit out-commented code.

Reply:
Without vm options, tests and debug IDE would be run against K1 mode. It's ok for 242 and below but for the 243+, it should be changed to something else. It's better to separate migration of the plugin and migration of the test infra indeed. Hopefully, it would be easy for the maintainers to find corresponding options without the comments.